mobile/ios/device/ios-application-manager.ts

```diff
@@ -92,8 +93,13 @@ export class IOSApplicationManager extends ApplicationManagerBase {
 	}
 
 	public async restartApplication(applicationId: string, appName?: string): Promise<void> {
-		await this.stopApplication(applicationId, appName);
-		await this.runApplicationCore(applicationId);
+		try {
+			await this.stopApplication(applicationId, appName);
+			await this.runApplicationCore(applicationId);
+		} catch (err) {
+			await this.$iOSNotificationService.postNotification(this.device.deviceInfo.identifier, `${applicationId}:NativeScript.LiveSync.RestartApplication`);
```

Comment:
This is handled in `appbuilder` subfolder in `ios-livesync-service.ts`. This change is irrelevant regarding Progress Telerik Platform and Cordova, as can be seen in the [file itself](https://github.com/telerik/mobile-cli-lib/blob/master/appbuilder/services/livesync/ios-livesync-service.ts#L26).